fendermint/vm/topdown/src/finality/null.rs

```diff
@@ -12,6 +12,8 @@ use ipc_sdk::staking::StakingChangeRequest;
 /// Finality provider that can handle null blocks
 #[derive(Clone)]
 pub struct FinalityWithNull {
+    /// The min topdown proposal height interval
```

Comment:
```suggestion
    /// The topdown proposal interval
```

Reply:
Why do we need a minimum? We should have a maximum, not a minimum. If you have caught up with the parent Lotus chain, and it proposes a block every 30 seconds, why could we not propose every single one?

Or what is this a minimum of? The number of blocks between proposals on the child subnet?

Reply:
I've been thinking a lot about this, and I agree with @aakoshh here. In the current model this offers us nothing, you can easily provide every non-null height and the operation should be the same.